src/components/Plugin/SafeSnap/Modal/OptionApproval.vue

```diff
@@ -10,13 +14,38 @@ export default {
     'questionId',
     'minimumBond',
     'tokenSymbol',
-    'tokenDecimals'
+    'tokenDecimals',
+    'oracle'
   ],
   emits: ['close', 'setApproval'],
+  setup() {
+    const { safesnap } = useSafesnap();
+    const { isValidEnsDomain } = useEns();
+    const debounce = useDebounce();
+    return { safesnap, debounce, isValidEnsDomain };
+  },
+  data() {
+    return { criteriaLink: '' };
+  },
+  mounted() {
+    this.debounce(this.getCriteriaLink);
```

Comment:
Calling `debounce` only once is not really it's purpose. If it's just about the initial delay, I'd use `setTimeout` here directly. But I think `criteriaLink` can just be a computed property and doesn't need any delaying.